sklearn/inspection/_plot/partial_dependence.py

```diff
@@ -274,35 +289,39 @@ def convert_feature(fx):
                              'len(feature_names) = {0}, got {1}.'
                              .format(len(feature_names), i))
 
-    # compute averaged predictions
+    # compute predictions and/or averaged predictions
     pd_results = Parallel(n_jobs=n_jobs, verbose=verbose)(
         delayed(partial_dependence)(estimator, X, fxs,
                                     response_method=response_method,
                                     method=method,
                                     grid_resolution=grid_resolution,
-                                    percentiles=percentiles)
+                                    percentiles=percentiles,
+                                    individual=individual)
         for fxs in features)
 
     # For multioutput regression, we can only check the validity of target
     # now that we have the predictions.
     # Also note: as multiclass-multioutput classifiers are not supported,
     # multiclass and multioutput scenario are mutually exclusive. So there is
     # no risk of overwriting target_idx here.
-    avg_preds, _ = pd_results[0]  # checking the first result is enough
-    if is_regressor(estimator) and avg_preds.shape[0] > 1:
+    pd_result, _ = pd_results[0]  # checking the first result is enough
+    n_tasks = pd_result[0].shape[0] if individual == 'both' \
```

Comment:
we tend to avoid `\` you can use paranthesis

```python
n_tasks = (pd.result[0].shape[0] if individual == "both"
           else pd.result.shape[0])
```